webui/react/src/ioTypes.ts

```diff
@@ -118,9 +118,9 @@ export type ioTypeMetric = io.TypeOf<typeof ioMetric>;
 const ioMetricSummary = io.type({
   count: optional(io.union([io.number, io.undefined])),
   last: optional(io.union([io.number, io.string, io.boolean])),
-  max: optional(io.number),
-  min: optional(io.number),
-  sum: optional(io.number),
+  max: optional(io.union([io.number, io.literal('Infinity'), io.literal('-Infinity'), io.literal('NaN')])),
```

Comment:
What you should actually be doing here is writing a custom decoder that maps those strings to the matching float values.

Reply:
Something like

```typescript
export class Float extends Type<number> {
  /**
   * @since 1.0.0
   */
  readonly _tag: 'FloatType' = 'FloatType'
  constructor() {
    super(
      'float',
      (u): t.number.is(u)
      (u, c) => {
        let u_ = u
        if (u === 'Infinity') {
          u_ = Infinity
        }
        ...
        return t.number.validate(u_, c);
      },
      (f) => {
        if (f === Infinity) {
          return 'Infinity'
        }
        ...
      },
    )
  }
}
```